package.json

```diff
@@ -23,6 +23,7 @@
 	"bin": {
 		"node-schematron": "./bin/schematron.js"
 	},
+	"main": "dist",
```

Comment:
Could you change this to `dist/index.js`?

Reply:
I'd be happy to, although there's actually no need to as `index.js` is the default.

Reply:
Done 🤝